Maven install throw exception

When run `mvn install` it throw exception,  but install result is success:

```
Tests run: 5, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.063 sec - in org.apache.shardingsphere.elasticjob.cloud.executor.TaskExecutorThreadTest
[ERROR] 2020-07-13 00:29:32,953 --test_job@-@0@-@READY@-@fake_slave_id@-@0_Worker-1-- [org.quartz.core.JobRunShell] Job DEFAULT.test_job threw an unhandled Exception:  
java.lang.IllegalStateException: Shutdown in progress
	at java.lang.ApplicationShutdownHooks.add(ApplicationShutdownHooks.java:66)
	at java.lang.Runtime.addShutdownHook(Runtime.java:211)
	at com.google.common.util.concurrent.MoreExecutors$Application.addShutdownHook(MoreExecutors.java:223)
	at com.google.common.util.concurrent.MoreExecutors$Application.addDelayedShutdownHook(MoreExecutors.java:195)
	at com.google.common.util.concurrent.MoreExecutors$Application.getExitingExecutorService(MoreExecutors.java:179)
	at com.google.common.util.concurrent.MoreExecutors$Application.getExitingExecutorService(MoreExecutors.java:214)
	at com.google.common.util.concurrent.MoreExecutors.getExitingExecutorService(MoreExecutors.java:148)
	at org.apache.shardingsphere.elasticjob.infra.concurrent.ElasticJobExecutorService.createExecutorService(ElasticJobExecutorService.java:52)
	at org.apache.shardingsphere.elasticjob.infra.handler.threadpool.impl.AbstractJobExecutorServiceHandler.createExecutorService(AbstractJobExecutorServiceHandler.java:32)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:75)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:63)
	at org.apache.shardingsphere.elasticjob.cloud.executor.DaemonTaskScheduler$DaemonJob.execute(DaemonTaskScheduler.java:182)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
[ERROR] 2020-07-13 00:29:32,955 --test_job@-@0@-@READY@-@fake_slave_id@-@0_Worker-1-- [org.quartz.core.ErrorLogger] Job (DEFAULT.test_job threw an exception. 
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: java.lang.IllegalStateException: Shutdown in progress
	at java.lang.ApplicationShutdownHooks.add(ApplicationShutdownHooks.java:66)
	at java.lang.Runtime.addShutdownHook(Runtime.java:211)
	at com.google.common.util.concurrent.MoreExecutors$Application.addShutdownHook(MoreExecutors.java:223)
	at com.google.common.util.concurrent.MoreExecutors$Application.addDelayedShutdownHook(MoreExecutors.java:195)
	at com.google.common.util.concurrent.MoreExecutors$Application.getExitingExecutorService(MoreExecutors.java:179)
	at com.google.common.util.concurrent.MoreExecutors$Application.getExitingExecutorService(MoreExecutors.java:214)
	at com.google.common.util.concurrent.MoreExecutors.getExitingExecutorService(MoreExecutors.java:148)
	at org.apache.shardingsphere.elasticjob.infra.concurrent.ElasticJobExecutorService.createExecutorService(ElasticJobExecutorService.java:52)
	at org.apache.shardingsphere.elasticjob.infra.handler.threadpool.impl.AbstractJobExecutorServiceHandler.createExecutorService(AbstractJobExecutorServiceHandler.java:32)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:75)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.<init>(ElasticJobExecutor.java:63)
	at org.apache.shardingsphere.elasticjob.cloud.executor.DaemonTaskScheduler$DaemonJob.execute(DaemonTaskScheduler.java:182)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
```

--- a/elasticjob-cloud/elasticjob-cloud-executor/src/main/java/org/apache/shardingsphere/elasticjob/cloud/executor/DaemonTaskScheduler.java
+++ b/elasticjob-cloud/elasticjob-cloud-executor/src/main/java/org/apache/shardingsphere/elasticjob/cloud/executor/DaemonTaskScheduler.java
@@ -160,6 +160,26 @@ public final class DaemonTaskScheduler {
         @Setter
         private Protos.TaskID taskId;
         
+        private volatile ElasticJobExecutor jobExecutor;
+        
+        private ElasticJobExecutor getJobExecutor() {
+            if (null == jobExecutor) {
+                createJobExecutor();
+            }
+            return jobExecutor;
+        }
+        
+        private synchronized void createJobExecutor() {
+            if (null != jobExecutor) {
+                return;
+            }
+            if (null == elasticJob) {
+                jobExecutor = new ElasticJobExecutor(elasticJobType, jobFacade.loadJobConfiguration(true), jobFacade);
+            } else {
+                jobExecutor = new ElasticJobExecutor(elasticJob, jobFacade.loadJobConfiguration(true), jobFacade);
+            }
+        }
+        
         @Override
         public void execute(final JobExecutionContext context) {
             ShardingContexts shardingContexts = jobFacade.getShardingContexts();
@@ -168,19 +188,11 @@ public final class DaemonTaskScheduler {
             if (jobEventSamplingCount > 0 && ++currentJobEventSamplingCount < jobEventSamplingCount) {
                 shardingContexts.setCurrentJobEventSamplingCount(currentJobEventSamplingCount);
                 jobFacade.getShardingContexts().setAllowSendJobEvent(false);
-                if (null == elasticJob) {
-                    new ElasticJobExecutor(elasticJobType, jobFacade.loadJobConfiguration(true), jobFacade).execute();
-                } else {
-                    new ElasticJobExecutor(elasticJob, jobFacade.loadJobConfiguration(true), jobFacade).execute();
-                }
+                getJobExecutor().execute();
             } else {
                 jobFacade.getShardingContexts().setAllowSendJobEvent(true);
                 executorDriver.sendStatusUpdate(Protos.TaskStatus.newBuilder().setTaskId(taskId).setState(Protos.TaskState.TASK_RUNNING).setMessage("BEGIN").build());
-                if (null == elasticJob) {
-                    new ElasticJobExecutor(elasticJobType, jobFacade.loadJobConfiguration(true), jobFacade).execute();
-                } else {
-                    new ElasticJobExecutor(elasticJob, jobFacade.loadJobConfiguration(true), jobFacade).execute();
-                }
+                getJobExecutor().execute();
                 executorDriver.sendStatusUpdate(Protos.TaskStatus.newBuilder().setTaskId(taskId).setState(Protos.TaskState.TASK_RUNNING).setMessage("COMPLETE").build());
                 shardingContexts.setCurrentJobEventSamplingCount(0);
             }

